src/region/RegionAnnotations.tsx

```diff
@@ -58,6 +59,12 @@ export default class RegionAnnotations extends React.PureComponent<Props, State>
     };
 
     handleChange = (text?: string): void => {
+        const { setCursorPosition } = this.props;
+
+        if (!text) {
+            setCursorPosition(0);
```

Comment:
@mxiao6, this method is already available in ReplyField. Can it be called from there? The parent component shouldn't need to know about the editor's cursor.